src/update/token_list.py

```diff
@@ -44,6 +46,6 @@ def update_token_list(dune: DuneAPI, token_list: list[str]) -> list[dict[str, st
     return results
 
 
-if __name__ == "__main__":
-    dune_connection = generic_script_init(description="Update Token List").dune
-    update_token_list(dune_connection, fetch_trusted_tokens())
+# if __name__ == "__main__":
```

Comment:
Can you explain this?

Reply:
For more context. During this refactor I was experimenting with "reexports" using the __init__ file. 

The idea was that long chains of redundant imports like this:

```
from src.fetch.period_slippage import get_period_slippage
from src.fetch.reward_targets import get_vouches
from src.fetch.eth_spent import get_eth_spent
from src.fetch.cow_rewards import get_cow_rewards
```

Could be expressed as:

```
from src.fetch import (
    get_period_slippage,
    get_vouches,
    get_eth_spent,
    get_cow_rewards,
)
```

as long as `__init__.py` looks like this:

```
from .period_slippage import get_period_slippage
from .reward_targets import get_vouches
from .eth_spent import get_eth_spent
from .cow_rewards import get_cow_rewards
```

Unfortunately this resulted in too many circular imports that I could not untangle (at least not now).


Glad you asked!